modules/sdk/src/test/scala/org/tessellation/sdk/infrastructure/snapshot/WeightedProspectSuite$.scala

```diff
@@ -0,0 +1,53 @@
+package org.tessellation.sdk.infrastructure.snapshot
+
+import cats.effect.std.Random
+
+import org.tessellation.schema.trust.TrustValueRefined
+
+import eu.timepit.refined.auto._
+import weaver.SimpleIOSuite
+import weaver.scalacheck.Checkers
+
+object WeightedProspectSuite$ extends SimpleIOSuite with Checkers {
```

Comment:
typo `$`?

Reply:
Renamed the file to `WeightedProspectSuite`.